pdns/recursordist/docs/settings.rst

```diff
@@ -362,7 +371,7 @@ Any servers' name suffix-matching the supplied names will never be throttled.
 ----------------------------
 .. versionadded:: 4.2.0
 
--  Comma separated list of netmasks
+-  Comma separated list of netmasks, negation not supported
```

Comment:
what makes `dont-throttle-netmasks` so special that negation is not supported?

Reply:
Ah, right. Perhaps that can go?

Reply:
LIkely yes, but I wanted docs only atm.